qualification/antenna_channelised_voltage/test_delay.py

```diff
@@ -240,23 +241,23 @@ async def test_delay_sensors(
         delay_strs.append(f"{delay},{delay_rate}:{phase},{phase_rate}")
         delay_tuples.append((load_ts, delay, delay_rate, phase, phase_rate))
 
-    def delay_sensor_value(label: str) -> tuple:
-        return literal_eval(cbf.sensors[f"antenna-channelised-voltage.{label}.delay"].value.decode())
+    async def delay_sensor_value(label: str) -> tuple:
+        return literal_eval(await pcc.sensor_value(f"antenna-channelised-voltage.{label}.delay", str))
```

Comment:
Good call (why was this blocking anyway?).

Reply:
Previously the sensor watcher pushed updates into the internal mirror of the sensors - so when you wanted the latest value you just looked at what was in the mirror. Switching from a push model to a pull model means this has to be asynchronous now because there is a request/reply cycle.